### ISSUE TITLE
StartObserverChangeswithCallback gets fired multiple time even for a single change in address book

### ISSUE THREAD

**Issue by techwizardg** [Alterplay/APAddressBook#160]
We are facing an issue .Our requirement is that when ever there is change in Phone book we need to listen  for it and act on it based on last modified date. When i perform a change in Native phone book (i.e. one change) the call back is getting fired multiple times. Are we doing something wrong?

Is our understanding right that the call back will get called only once for a change

**Comment by belkevich** [comment]
Yes, it should be called only once. Try `Example` app with your device. Does it call multiple times?
Also, please, provide some code.

**Comment by techwizardg** [comment]
The example app also gets called multiple times. Would provide snippets soon

**Comment by ortizroberto** [comment]
I am facing the same issue, even if I go to the Contacts App and don't change anything when I get back  `startObserveChanges` gets fired.
